src/workloads/scale/MemoryStress.yml

```diff
@@ -0,0 +1,90 @@
+SchemaVersion: 2018-07-01
+Owner: Service Architecture
+Description: |
+  Used to test performance of the server under heavy memory pressure.
+
+GlobalDefaults:
+  dbname: &DBName memorystress
+  MaxPhases: &MaxPhases 20
+
+  LoadThreads: &LoadThreads 128
+  LoadBatchSize: &LoadBatchSize 500
+
+  NumberOfDocuments: &NumDocs 20096
+  # NumberOfDocumentsPerWorker = NumberOfDocuments / LoadThreads.
+  NumberOfDocumentsPerWorker: &DocsPerThread 157
+  DocSize: &DocSize 520000
+  Document: &Doc
+    # Document number/ID.
+    a: {^Inc: {start: -157, step: 1, multiplier: *DocsPerThread}}
```

Comment:
This is an unfortunate result of the equation for `^Inc = start + ActorId * multiplier + i * step`. I think ActorId starts at 1,  therefore in order for the true start of `^Inc` to be 0, it would mean `0 = start + 1*157 + 0*1` -> `start=-157`. I tried to use the preprocessor to do something like `start:-*DocsPerThread`, but that doesn't seem to pass `run-genny evaluate`.

I can put a comment here describing the above, or I could start the [sort at line 77](https://github.com/mongodb/genny/blob/e301a291f620b10d547e23a050f9c874597d021d/src/workloads/scale/MemoryStress.yml#L77) at `*DocSize` and end at `*DocSize + *NumDocs`. lmk what you think

Reply:
Done!